include/dbus_utility.hpp

```diff
@@ -110,5 +126,128 @@ inline void checkDbusPathExists(const std::string& path, Callback&& callback)
         std::array<std::string, 0>());
 }
 
+template <std::size_t N>
+inline void
+    getSubTree(const std::string& path, int32_t depth,
+               const std::array<const char*, N>& interfaces,
```

Comment:
We are using std::array instead of std::span, std::span is not available in 1030?

Reply:
@Lakshmi-y 

1030 code base does not fully support `std::span<std::string_view> `or `std::span<const char*>`.
Using 1050 or upstream coding style will fail to compile in 1030.

So, I made the code as closest as possible to 1050 or upstream style. using `<template std::size_t N> `and `const std::array<const char*, N>`.

With those, codes between 1030 & 1050 become more or less the same - except using `const char*` instead of `std::string_view`.